test/auth.test.ts

```diff
@@ -62,15 +61,6 @@ describe('Google Authentication', () => {
     done();
   });
 
-  it('generate token', done => {
-    const generatedToken = userToken(
-      '92c472c8-406a-4a89-898f-46965830316a',
-      'nadinefiona9@gmail.com'
-    );
-    expect(typeof generatedToken).to.equal('string');
-    done();
-  });
-
   it('should handle Google authentication failure', done => {
```

Comment:
why removing this test it's shouldn't be removed

Reply:
that's what we have on develop branch you can even check yourself i didnt touch this test!!!